testdata/watcher_interval/src/a

```diff
@@ -0,0 +1 @@
+test
```

Comment:
I feel that this file shouldn't be commited? We usually have a `.gitignore` inside each directory inside `testdata`.